src/app/components/layout/AspectRatio.js

```diff
@@ -146,13 +146,13 @@ const AspectRatio = ({
           tagName="p"
         />
       ) : null }
-      <div style={{ visibility: contentWarning && maskContent && !superAdminMask ? 'visible' : 'hidden' }}>
+      {contentWarning && maskContent && !superAdminMask ?
```

Comment:
not sure if there was a reason why the previous implementation was this way, but using visibility keeps the div as part of the dom so it affects layout even though its not seen.